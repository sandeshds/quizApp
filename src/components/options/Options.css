.optionsBlock {
    margin-bottom: 20px;
    position: absolute;
    bottom:20px;
    left:40%;
}

.button {
    margin-right: 5px;
    background: transparent;
    border: 1px solid white;
    color: white;
    height: 50px;
    width: 70px;
    font-size: large;
    outline: none;
}