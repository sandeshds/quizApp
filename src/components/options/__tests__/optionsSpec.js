import React, {Component} from 'react';
import renderer from 'react-test-renderer';
import Options from './../index';

describe('Options', ()=>{
    it('should render options properly', ()=>{
        let tree = renderer.create(
            <Options onSelectingAnswer={() => {}}/>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});