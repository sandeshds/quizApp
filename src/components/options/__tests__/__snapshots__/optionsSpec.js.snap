// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Options should render options properly 1`] = `
<div
  className="optionsBlock"
>
  <input
    className="button"
    onClick={[Function]}
    type="button"
    value="A"
  />
  <input
    className="button"
    onClick={[Function]}
    type="button"
    value="B"
  />
  <input
    className="button"
    onClick={[Function]}
    type="button"
    value="C"
  />
  <input
    className="button"
    onClick={[Function]}
    type="button"
    value="D"
  />
</div>
`;
